### PR TITLE
MRG: Small fix to tutorial; rename plot_events ordinate label to "Event id"; improve some SSP docstrings

### PR DESCRIPTION
**Pull request by hoechenberger**
- ECG revents correspond to R wave peaks
- Ordinate label should be in singular, i.e. "Event id" instead of "Events id" – unless we want consistency with the `events_id` dict, but then it should be called "events_id"… which doesn't make too much sense to me
- improve some SSP docstrings

### CODE REVIEW

**Review by larsoner — APPROVED** [review]
LGTM but style is unhappy

File: tutorials/preprocessing/plot_50_artifact_correction_ssp.py [hunk 1]
```diff
@@ -228,8 +228,8 @@
 # projectors for magnetometers, gradiometers, and EEG channels (default is two
 # projectors for each channel type).
 # :func:`~mne.preprocessing.compute_proj_ecg` also returns an :term:`events`
-# array containing the sample numbers corresponding to the onset of each
-# detected heartbeat.
+# array containing the sample numbers corresponding to the peak of the R wave
```
Comment (drammock) [inline tutorials/preprocessing/plot_50_artifact_correction_ssp.py:231]: can we link "the R wave" to something here?  Maybe https://en.wikipedia.org/wiki/QRS_complex ?

**Review by drammock — APPROVED** [review]
looks good! just one suggestion

File: tutorials/preprocessing/plot_50_artifact_correction_ssp.py (hunk 1 above)
Comment (hoechenberger) [reply tutorials/preprocessing/plot_50_artifact_correction_ssp.py:231]: Great idea! Will do.

File: mne/preprocessing/ssp.py [hunk 2]
```diff
@@ -150,9 +150,19 @@ def compute_proj_ecg(raw, raw_event=None, tmin=-0.2, tmax=0.4,
...
-    .. note:: raw data will be loaded if it is not already.
+    This function will:
+
+      - filter the ECG data channel,
+      - find ECG R wave peaks using :func:`mne.preprocessing.find_ecg_events`,
+      - filter the raw data,
+      - create `~mne.Epochs` around the R wave peaks, capturing the heartbeats,
+      - optionally average the `~mne.Epochs` to produce an `~mne.Evoked` if
+        ``average=True`` was passed (default), and finally
+      - calculate SSP projection vectors on that data to capture the artifacts.
```
Comment (drammock) [inline mne/preprocessing/ssp.py:163]: these are indented too far, they're getting rendered as both bullets and blockquotes.

https://23991-1301584-gh.circle-artifacts.com/0/dev/generated/mne.preprocessing.compute_proj_ecg.html#mne.preprocessing.compute_proj_ecg

File: mne/preprocessing/ssp.py [hunk 3]
```diff
@@ -263,9 +273,20 @@ def compute_proj_eog(raw, raw_event=None, tmin=-0.2, tmax=0.2,
...
+    This function will:
+
+      - filter the EOG data channel,
+      - find the peaks of eyeblinks in the EOG data using
+        :func:`mne.preprocessing.find_eog_events`,
+      - filter the raw data,
+      - create `~mne.Epochs` around the eyeblinks,
+      - optionally average the `~mne.Epochs` to produce an `~mne.Evoked` if
+        ``average=True`` was passed (default), and finally
+      - calculate SSP projection vectors on that data to capture the artifacts.
```
Comment (drammock) [inline mne/preprocessing/ssp.py:287]: indentation again: https://23991-1301584-gh.circle-artifacts.com/0/dev/generated/mne.preprocessing.compute_proj_eog.html

File: mne/preprocessing/ecg.py [hunk 4]
```diff
@@ -177,8 +164,12 @@ def find_ecg_events(raw, event_id=999, ch_name=None, tstart=0.0,
         The events corresponding to the peaks of the R waves.
     ch_ecg : string
         Name of channel used.
-    average_pulse : float
-        Estimated average pulse.
+    average_pulse : float | np.nan
+        The estimated average pulse. If no ECG events could be found, this will
+        be ``nan``.
```
Comment (drammock) [inline mne/preprocessing/ecg.py:169]: is this accurate?  Do we ever return `np.nan`? It looks like the part of the PR that changed that in the code has been removed.

Comment (hoechenberger) [reply mne/preprocessing/ecg.py]: Good catch. I have corrected this.

File: mne/preprocessing/ssp.py (hunk 2 above)
Comment (hoechenberger) [reply mne/preprocessing/ssp.py]: I have fixed that, and also changed the list to an enumeration.

**Comment by hoechenberger** [comment]
@drammock I think this is good to merge.

Doc artifacts:
- [`create_ecg_epochs`](https://24025-1301584-gh.circle-artifacts.com/0/dev/generated/mne.preprocessing.create_ecg_epochs.html)
- [`create_eog_epochs`](https://24025-1301584-gh.circle-artifacts.com/0/dev/generated/mne.preprocessing.create_eog_epochs.html)
- [`compute_proj_ecg`](https://24025-1301584-gh.circle-artifacts.com/0/dev/generated/mne.preprocessing.compute_proj_ecg.html)
- [`compute_proj_eog`](https://24025-1301584-gh.circle-artifacts.com/0/dev/generated/mne.preprocessing.compute_proj_eog.html)

**Comment by drammock** [comment]
Thx @hoechenberger !

**Pull request merged**